### PR TITLE
add August update

### PR DESCRIPTION
**Pull request by dholbach**
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

File: content/en/blog/2022-09-05-august-update/index.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+{{< /imgproc >}}
+
+#### Weave GitOps
+
+[https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.2](https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.2)
+
+[https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.3](https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.3)
+
+[https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.4](https://github.com/weaveworks/weave-gitops/releases/tag/v0.9.4)
+
```
Comment (dholbach) [inline content/en/blog/2022-09-05-august-update/index.md:129]: ping @ozamosi @chanwit @JamWils

**Comment by dholbach** [comment]
→ https://deploy-preview-1132--fluxcd.netlify.app/blog/

File: content/en/blog/2022-09-05-august-update/index.md (hunk 1 above)
Comment (dholbach) [reply content/en/blog/2022-09-05-august-update/index.md:131]: Ok. I'll just add a small note that a number of releases came out and that people should upgrade and consult the changelog.

**Review by stefanprodan — APPROVED** [review]
LGTM 

Thanks @dholbach

**Pull request merged**